packages/documentation/gatsby-config.js

```diff
@@ -1,6 +1,3 @@
-/* eslint-disable */
-const path = require('path');
```

Comment:
I'm assuming `path` wasn't used. If so, I'm surprised that wasn't caught by a linter.

After [searching the repo for `eslintrc`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/search?q=eslintrc&unscoped_q=eslintrc), it looks like we actually [ignore `.eslintrc`](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/e0049086edb435e5a7881360504352269d7b05d0/.gitignore#L12), and unignore it only within the [`.gitignore` file for the `vagov-eslint` package](https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/blob/e0049086edb435e5a7881360504352269d7b05d0/packages/vagov-eslint/.gitignore).

Reply:
There are several places I've found that didn't seem to actually get linted or formatted by Prettier. That checks out...